warehouse/models/mart/ntd_annual_reporting/fct_capital_expenses_by_capital_use.sql

```diff
@@ -3,9 +3,25 @@ WITH staging_capital_expenses_by_capital_use AS (
     FROM {{ ref('stg_ntd__capital_expenses_by_capital_use') }}
 ),
 
+current_dim_organizations AS (
+    SELECT
+        ntd_id,
+        caltrans_district
+    FROM {{ ref('dim_organizations') }}
+    WHERE _is_current
```

Comment:
Since I'm seeing this clause added in lots of places, can we either make a fix so that it is date-bound or add a note about this in the documentation?

Reply:
See next comment. This should be an intermediate table and the documentation or date-bounding could be added to that intermediate table.